unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWInputHandler.cs

```diff
@@ -0,0 +1,218 @@
+using DCL.Controllers;
+using DCL.Helpers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BIWInputHandler : BIWController
+{
+    [Header("Design variables")]
+    public float msBetweenInputInteraction = 200;
+
+    [Header("References")]
+    public BuilderInWorldController builderInWorldController;
+    public ActionController actionController;
+    public BIWModeController biwModeController;
+    public BuilderInWorldInputWrapper builderInputWrapper;
+    public BIWOutlinerController outlinerController;
+    public BuilderInWorldEntityHandler builderInWorldEntityHandler;
```

Comment:
Unless we need them to be public, we should make them `[SerializeField] private`